Detectors/FIT/base/include/FITBase/Digit.h

```diff
@@ -0,0 +1,67 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FIT_DIGIT_H_
+#define ALICEO2_FIT_DIGIT_H_
+
+#include "CommonDataFormat/TimeStamp.h"
+#include <iosfwd>
+#include "Rtypes.h"
+
+#include <boost/serialization/base_object.hpp> // for base_object
```

Comment:
this include is likely not needed. Can you please remove it?

Reply:
Yes, last one is not need now.